Configuration/AlCa/python/autoAlca.py

```diff
@@ -34,7 +34,7 @@
                   "DoubleMu" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlZMuMu",
                   "DoubleMuParked" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlZMuMu",
                   "MuOniaParked" : "TkAlJpsiMuMu+TkAlUpsilonMuMu",
-                  "DoubleElectron" : "EcalCalZElectron+EcalUncalZElectron+HcalCalIsoTrkFilter",
+                  "DoubleElectron" : "EcalCalZElectron+EcalUncalZElectron",
```

Comment:
No replacement here, simply removed?

Reply:
@bsunanda

Reply:
Yes. We do not need this AlCaReco stream for IsoTrack. We only need this from JetHT and DoubleEgamma